73_Mortons/CTester.h

```diff
@@ -0,0 +1,399 @@
+#ifndef _NBL_EXAMPLES_TESTS_12_MORTON_C_TESTER_INCLUDED_
+#define _NBL_EXAMPLES_TESTS_12_MORTON_C_TESTER_INCLUDED_
+
+#include <nabla.h>
+#include "app_resources/testCommon.hlsl"
+#include "ITester.h"
```

Comment:
why are we reusing things from a different example ?

if you want `ITester` to be general, please move to appropriate directory like `common/include/nbl/examples/testing`

see https://github.com/Devsh-Graphics-Programming/Nabla-Examples-and-Tests/tree/08c898d5af460ba6469a78fb625216e27a1bc8a8/common/include/nbl/examples

Reply:
assigned @Przemog1 in next PR